docs/pages/project/contributing/contributing-cli.md

```diff
@@ -128,6 +128,27 @@ When designing for the command line interface, ask and consider the following qu
 
 You will need to provide a short and long description of the command for the help pages and also for the Meshery Documentation.
 
+## Refactoring Mesheryctl Code
+
+In the latest update of Meshery, the meshconfig has been refactored to have getters and setters which can further be used to manipulate any config, making it more easier to perform specific functions. 
```

Comment:
"In the latest update of Meshery, "... are we asking contributors to work on old versions of mesheryctl? What relevance does the latest update have in contributor instructions? This isn't a blog post. These are timeless instructions.